### DISCUSSION TITLE
[BUG] Inline math rendering failing sporadically

### DISCUSSION THREAD

**Discussion by AaronYong179** [community/community#30747] [Other Feature Feedback, Questions, & Ideas]
Math rendering seems to only occur correctly for one line, then fails to work for subsequent lines. 

Here is an example from the [blog post on math support](https://github.blog/2022-05-19-math-support-in-markdown/)

When $a \ne 0$, there are two solutions to $(ax^2 + bx + c = 0)$ and they are 
$$ x = {-b \pm \sqrt{b^2-4ac} \over 2a} $$

The bottom line doesn't render correctly at all.

These: $a \ne 0$ and $(ax^2 + bx + c = 0)$ do not seem to work either.

**Comment by nschloe** [comment]
A workaround here is to remove the spaces around the double dollars. It's still a bug though. A bit unfortunate that this happens for their _announcement example_.
